src/integer.h

```diff
@@ -17,235 +17,8 @@
 
 #ifdef USE_GMP /* then use the gmp version of the header file */
  #include "gmpints.h"
-#else /* read the rest of this file */
-
-
-/****************************************************************************
-**
-
-*T  TypDigit  . . . . . . . . . . . . . . . . . . . .  type of a single digit
-**
-**  'TypDigit' is the type of a single digit of an  arbitrary  size  integer.
-**  This is of course unsigned short int, which gives us the 16 bits we want.
-*/
-#ifdef SYS_IS_64_BIT
-typedef UInt4           TypDigit;
-#define INTEGER_UNIT_SIZE 4
-#define INTEGER_ALLOCATION_SIZE 16
 #else
-typedef UInt2           TypDigit;
-#define INTEGER_UNIT_SIZE 2
-#define INTEGER_ALLOCATION_SIZE 8
+ #warning "GAP integer code has been removed."
```

Comment:
Minor pick, this may as well be #error, else users will (I think) end up with a link-time error.